src/main/java/org/jabref/gui/importer/fetcher/WebSearchPaneViewModel.java

```diff
@@ -19,6 +19,7 @@
 import org.jabref.logic.importer.ParserResult;
 import org.jabref.logic.importer.SearchBasedFetcher;
 import org.jabref.logic.importer.WebFetchers;
+import org.jabref.logic.importer.FetcherException;
```

Comment:
This should be reordered in alphabetical order.

Reply:
Sure. Thank your for your review.